includes/class-edd-license-handler.php

```diff
@@ -5,7 +5,7 @@
  * This class should simplify the process of adding license information
  * to new EDD extensions.
  *
- * @version 3.0
+ * @version 1.1
```

Comment:
Does it matter that this has changed down to 1.1?

Reply:
It was 1.1 in master. Looks like it had been changed to 3.0 in release/3.0. I kind of decided on the spot that the version was meant to designate the version of the class -- not the version of current EDD -- and no major changes had been made to really designate a massive version jump. :shrug: